test/onnx/test_fx_op_consistency.py

```diff
@@ -72,13 +71,13 @@
         "ceil", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Ceil")
     ),
-    fixme("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
+    skip("ceil", dtypes=[torch.float64], reason=onnx_test_common.reason_onnx_runtime_does_not_support("Ceil", ["f64"])),
```

Comment:
curious why is this skip instead of xfail?

Reply:
The way of how xfail vs skip is that we try our best to put on xfail, but if it's not consistently fail, for example, unexpectedly success on few cases, we have to use skip.

But for this one, it's a legacy fixme. Great catch!